pkg/server/server.go

```diff
@@ -76,17 +76,18 @@ var (
 		NativeHistogramMinResetDuration: time.Hour,
 	}, []string{"grpc_service", "grpc_method"})
 
+	//The name of the histogram is not referring to dispatch count, but we are not changing this as it will be a breaking change.
 	requestDurationByQueryHistogramName = "request_duration_by_query_count_ms"
 
-	requestDurationByQueryHistogram = promauto.NewHistogramVec(prometheus.HistogramOpts{
+	requestDurationByQueryAndDispatchHistogram = promauto.NewHistogramVec(prometheus.HistogramOpts{
 		Namespace:                       build.ProjectName,
 		Name:                            requestDurationByQueryHistogramName,
```

Comment:
The name of this metric has become too overloaded and is too specific and misleading. We should rename to `request_duration_ms` instead of `request_duration_by_query_count_ms`

Reply:
I discussed about this with @adriantam since it is breaking change. Ended up changing only the internal names. Should we go ahead with this?

Reply:
Let’s introduce it in a compatible way.

Release `v1.5.1`  we introduce `request_duration_ms` and announce deprecation notice of `request_duration_by_query_count_ms`.

* `request_duration_ms`
* `request_duration_by_query_count_ms`

These report the same values in release `v1.5.1`.

(wait 2-4 weeks)

Release `v1.5.X` we drop `request_duration_by_query_count_ms` in favor of `request_duration_ms`.

Reply:
Great, I will do this in another PR